smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -169,6 +177,25 @@ private String getCommandExample(String serviceName, String configName, String c
             + String.format("@see {@link %s | config} for %s's `config` shape.%n", configName, serviceName);
     }
 
+    private String getThrownExceptions() {
+        List<ShapeId> errors = operation.getErrors();
+        StringBuilder buffer = new StringBuilder();
+        for (ShapeId error : errors) {
+            Shape errorShape = model.getShape(error).get();
+            Optional<DocumentationTrait> doc = errorShape.getTrait(DocumentationTrait.class);
+            ErrorTrait errorTrait = errorShape.getTrait(ErrorTrait.class).get();
+
+            if (doc.isPresent()) {
+                buffer.append(String.format("@throws {@link %s} (%s fault) %s%n",
```

Comment:
Instead of using `%n` at the end of each of these lines, append a `"\n"` after the else statement.